packages/grafana-ui/src/components/Monaco/CodeEditor.tsx

```diff
@@ -125,6 +125,9 @@ class UnthemedCodeEditor extends React.PureComponent<Props> {
       tabSize: 2,
       codeLens: false,
       contextmenu: false,
+      glyphMargin: true,
+      lineNumbers: 'on',
+      folding: true,
```

Comment:
I don't think we should change these defaults without checking with lots of people. I think you should revert this change and go ahead and merge this.

Reply:
alright 👍